contracts/src/v0.8/ccip/test/pools/GHO/fork/GhoTokenPoolMigrate1_4To1_5/TokenPoolsUpgrade.t.sol

```diff
@@ -0,0 +1,123 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.19;
+
+import {ForkBase} from "./ForkBase.t.sol";
+import {UpgradeableLockReleaseTokenPool_Sepolia} from "./LegacyTestnetTokenPools/UpgradeableLockReleaseTokenPool_Sepolia.sol";
+import {UpgradeableBurnMintTokenPool_ArbSepolia} from "./LegacyTestnetTokenPools/UpgradeableBurnMintTokenPool_ArbSepolia.sol";
+import {TransparentUpgradeableProxy} from "solidity-utils/contracts/transparent-proxy/TransparentUpgradeableProxy.sol";
+import {ProxyAdmin} from "solidity-utils/contracts/transparent-proxy/ProxyAdmin.sol";
+import {Client} from "../../../../../libraries/Client.sol";
+import {Internal} from "../../../../../libraries/Internal.sol";
+
+contract ForkPoolUpgradeAfterMigration is ForkBase {
+  function setUp() public override {
+    super.setUp();
+
+    // #1: deploy new implementation & upgrade token pools
+    vm.selectFork(l1.forkId);
+    _upgradeExistingLockReleaseTokenPool();
+
+    vm.selectFork(l2.forkId);
+    _upgradeExistingBurnMintTokenPool();
+
+    // #2: setProxyPool
+    vm.selectFork(l1.forkId);
+    vm.prank(l1.tokenPool.owner());
+    l1.tokenPool.setProxyPool(l2.chainSelector, l1.proxyPool);
+
+    vm.selectFork(l2.forkId);
+    vm.prank(l2.tokenPool.owner());
+    l2.tokenPool.setProxyPool(l1.chainSelector, l2.proxyPool);
+  }
+
+  function test_sendFromLegacyRouterL1() public {
```

Comment:
adjust function names to get aligned with others

Reply:
[5153d5d](https://github.com/aave/ccip/pull/18/commits/5153d5da1960313b95218e97d170619c72a2b985)